Libplanet.Explorer.Executable/Program.cs

```diff
@@ -93,7 +94,7 @@ internal static int Parse(string[] args)
             {
                 stderr.WriteLine("error: {0}", e.Message);
                 stderr.WriteLine(
-                    "Try `{0}' --help' for more information.",
+                    "Try `{0} --help' for more information.",
```

Comment:
```
`{0} --help'
```
 starts with `` ` `` and ends with `'`. Is there a reason?

Reply:
AFAIK it is a common style in Unix command-line programs, since there are no curly quotes in ASCII.  See also:

<https://en.wikipedia.org/wiki/Quotation_mark#Typewriters_and_early_computers>
<https://en.wikipedia.org/wiki/Quotation_mark#Curved_quotes_within_and_across_applications>